src/views/Gallery/Gallery.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Carousel from "react-elastic-carousel";
+import IMG1 from "../../asserts/img1.webp";
+import IMG2 from "../../asserts/img2.webp";
+import IMG3 from "../../asserts/img3.webp";
+import IMG4 from "../../asserts/img4.webp";
+import IMG5 from "../../asserts/img5.webp";
+import IMG6 from "../../asserts/img6.webp";
+import IMG7 from "../../asserts/img7.webp";
```

Comment:
Do not import images as files. It will reduce the performance of the application. Add all the images to the `assets` folder which is available in the `public` folder and use image as follows.
```
<img src="/assets/img1.webp" className="mr-10" />
```